components/service-mgt/axis2-service-mgt/org.wso2.carbon.service.mgt.ui/src/main/resources/web/service-mgt/checkForGroupedServices_ajaxprocessor.jsp

```diff
@@ -24,7 +24,7 @@
 <%@ page import="java.util.ResourceBundle" %>
 <%@ page import="org.wso2.carbon.ui.util.CharacterEncoder" %>
 <%
-    String serviceGroupsString = CharacterEncoder.getSafeText(request.getParameter("serviceGroupsString"));
```

Comment:
getSafeText() is removed to solve the problem which is occurred due to double encoding.Here "serviceGroupString" is not outputted and encoding for "serviceGroupString" should be happened in back-end side.